src/components/timeline/Day.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface DayProps {
+  children: React.ReactNode;
+  day: string;
+}
+
+const Day = ({ children, day }: DayProps): JSX.Element => {
+  return (
+    <StyledDay>
+      <DayString>{day}</DayString>
+      <div>{children}</div>
+    </StyledDay>
+  );
+};
+
+export default Day;
+
+const StyledDay = styled.div`
+  display: flex;
+`;
+
+const DayString = styled.h2`
+  color: #f9a11b;
+  font-weight: 400;
```

Comment:
Very nitpicky, but I'd prefer using `font-weight: normal`. For example, the keyword "bold" means 700, but uses [fallback weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Fallback_weights) if it isn't found. The specific font weight numbers we import could vary, so using keywords ensures you'll get "something bold-ish" even if specifically 700 isn't available.